LEAF_Nexus/api/controllers/EmployeeController.php

```diff
@@ -69,7 +69,7 @@ public function post($act)
         });
 
 
-        $this->index['POST']->register('employee/refresh/[text]/[digit]', function ($args) use ($employee, $national_db) {
+        $this->index['POST']->register('employee/refresh/[text]', function ($args) use ($employee, $national_db) {
```

Comment:
Is it possible that someone might be using this endpoint in custom code? If so then there call isn't going to find the endpoint they are pointing to.

Reply:
It's possible, but unlikely as this endpoint was only added last month.

To be sure, I've added it back and opened https://github.com/department-of-veterans-affairs/LEAF/issues/2062